modules/bibauthorid/lib/bibauthorid_daemon.py

```diff
@@ -190,6 +231,22 @@ def _task_run_core():
             run_tortoise(from_scratch)
             bibtask.task_update_progress('Full disambiguation finished!')
 
+    if bibtask.task_get_option("force_identifier_consistency"):
+        check_db = bibtask.task_get_option("check_db")
+        dry_run_b = bibtask.task_get_option("dry_run")
+        query_file_size = bibtask.task_get_option("packet_size")
+        dry_hepnames = bibtask.task_get_option("dry_hepnames_run")
+        tickets = bibtask.task_get_option("open_tickets")
+        queue = bibtask.task_get_option("queue")
+        bibtask.task_update_progress('Initializing hoover')
+        if query_file_size:
+            run_hoover(check_db_consistency=check_db, dry_run=dry_run_b, packet_size=query_file_size, dry_hepnames_run=dry_hepnames, open_tickets=tickets)
+        if queue:
+            run_hoover(check_db_consistency=check_db, dry_run=dry_run_b, dry_hepnames_run=dry_hepnames, open_tickets=tickets, queue=queue)
+        else:
```

Comment:
Note, with this set of `if..else` if one sets `query_file_size` he will execute hoover over the default `Test` queue and then a second time!